Design.md

```diff
@@ -227,8 +229,26 @@ This API will enable or disable user on Vault.
 
 
 ### Create S3 Credential
-This API creates S3 credential for the user.
-1. `generate-access-key` api will be called using assumed role credentials.
+This API creates S3 credentials for the user.
+1. The Generate Access Key API will be called using assumed role credentials.
+1. Encrypt the secret key (For more, see [SecretKey-Encryption-Strategy](#SecretKey-Encryption-Strategy)). Store the encrypted secret key on Redis Sentinel in the hash named "osis:s3credentials", with the hash key formatted as `<Username>__<AccessKeyID>`.
+1. Log the access key ID _(Log location and tools for logging are yet to be decided)_
+
+**Important Notes:**
+* Redis Sentinel is subject to an ongoing rolling deployment.
+* If Redis Sentinel fails or crashes, the vCloud Director's OSE service must be restarted with the following command:
```

Comment:
```suggestion
* If the entire Redis Sentinel cluster fails or crashes, the vCloud Director's OSE service must be restarted with the following command:
```

Reply:
fixed in e87f4cb37a396aa2237d83bf61f623d6fb24ccc4